gitops/azure-devops/build.sh

```diff
@@ -1,8 +1,14 @@
-#!/bin/bash
-
-function copy_files() {
+function init() {
     cp -r * $HOME/
     cd $HOME
+
+    if [[ "$GIT_TYPE" == "github" ]]; then
```

Comment:
@yradsmikham what is the result if not one of the handled conditions? What is $ACCESS_TOKEN is undefined. Perhaps we should have a fast fail in the beginning of the script if those variables are not defined or unexpected.

Reply:
Sure, I like that idea. Let me make that change and test it. Will get back to y’all.